src/main/resources/application.yml

```diff
@@ -23,7 +23,7 @@ spring:
 # Fake User #
 #############
 fake-user:
-  enabled: true
+  enabled: false
```

Comment:
```suggestion
  enabled: true
```

@MiniDigger Do you want this to still default to `true`? `false` means SpongeAuth is required.

Reply:
lets keep enabled for now, untill we figure out deployment for auth stuff